lookups/v1/phone_number/fetch-lookup-get-addon-payfone-example-1/fetch-lookup-get-addon-payfone-example-1.3.x.js

```diff
@@ -6,7 +6,7 @@ const client = require('twilio')(accountSid, authToken);
 
 client.lookups.phoneNumbers('+15108675310')
       .fetch({addOns: 'payfone_tcpa_compliance', addOnsData: {
-           "payfone_tcpa_compliance.right_party_contacted_date": "20160101"
+           'payfone_tcpa_compliance.right_party_contacted_date': 20160101
```

Comment:
Here's another instance of something that looks like a number is getting the quotes removed